<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrcIdentitiesEditDialog</class>
 <widget class="QDialog" name="IrcIdentitiesEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRC Identities</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Identity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="identitiesCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the identity to work on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="toolTip">
        <string>Press to add a new identity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="copyButton">
       <property name="toolTip">
        <string>Press to copy the selected identity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="renameButton">
       <property name="toolTip">
        <string>Press to rename the selected identity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="toolTip">
        <string>Press to delete the selected identity</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="identityTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Real Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="realnameEdit">
         <property name="toolTip">
          <string>Enter the real name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="nickNameGroup">
         <property name="title">
          <string>Nick Names</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="nicknameEdit">
            <property name="toolTip">
             <string>Enter a nick name to add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="5">
           <widget class="QListWidget" name="nicknamesList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="nicknameAddButton">
            <property name="toolTip">
             <string>Press to add the entered nick name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="nicknameDeleteButton">
            <property name="toolTip">
             <string>Press to delete the selected nick name</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QToolButton" name="nicknameUpButton">
            <property name="toolTip">
             <string>Press to move the selected nick name up</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QToolButton" name="nicknameDownButton">
            <property name="toolTip">
             <string>Press to move the selected nick name down</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>87</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="autoIdentifyGroup">
         <property name="title">
          <string>Auto Identify</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Service:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="serviceEdit">
            <property name="toolTip">
             <string>Enter the name of the service to identify against</string>
            </property>
            <property name="whatsThis">
             <string>Service name can be &lt;b&gt;&lt;i&gt;nickserv&lt;/i&gt;&lt;/b&gt; or a network-dependent name such as &lt;b&gt;&lt;i&gt;nickserv@services.dal.net&lt;/i&gt;&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordEdit">
            <property name="toolTip">
             <string>Enter the password</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="awayTab">
      <attribute name="title">
       <string>Away</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="rememberPosOnAwayCheckBox">
         <property name="toolTip">
          <string>Select to mark the current position in the chat, when you send an AWAY command.</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>Mark the current position in chat windows when going away</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Away Message:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="awayEdit">
         <property name="toolTip">
          <string>Enter the message to be sent when going away</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>219</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Ident:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="identEdit">
         <property name="toolTip">
          <string>Enter the identity to be used to log-on to the server</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Reason for Quit:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="quitEdit">
         <property name="toolTip">
          <string>Enter a message to be sent when quitting</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Reason for Part:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="partEdit">
         <property name="toolTip">
          <string>Enter message to be sent when leaving a channel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>291</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>identitiesCombo</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>renameButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>identityTabWidget</tabstop>
  <tabstop>realnameEdit</tabstop>
  <tabstop>nicknameEdit</tabstop>
  <tabstop>nicknamesList</tabstop>
  <tabstop>nicknameAddButton</tabstop>
  <tabstop>nicknameDeleteButton</tabstop>
  <tabstop>nicknameUpButton</tabstop>
  <tabstop>nicknameDownButton</tabstop>
  <tabstop>serviceEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>rememberPosOnAwayCheckBox</tabstop>
  <tabstop>awayEdit</tabstop>
  <tabstop>identEdit</tabstop>
  <tabstop>quitEdit</tabstop>
  <tabstop>partEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IrcIdentitiesEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IrcIdentitiesEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
